### PR TITLE
AirShip Enums + Example fillin

### PR DESCRIPTION
**Pull request by JohnTranQUT**
### Description
There are now 39 Tasks, was 40 in earlier versions, with #3 was unknown
Client must had changed.
Also added Example to Example Plugin for debugging purposes.

### CODE REVIEW

File: src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs [hunk 1]
```diff
@@ -84,6 +84,11 @@ public async ValueTask OnPlayerChat(IPlayerChatEvent e)
                 await e.PlayerControl.NetworkTransform.SnapToAsync(new Vector2(1, 1));
             }
 
+            if(e.Message == "completetasks")
```
Comment (Minorusama) [inline src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs]: Please fix spacing after `if`

File: src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs [hunk 2]
```diff
@@ -111,5 +116,11 @@ public void OnPlayerVentEvent(IPlayerVentEvent e)
         {
             _logger.LogInformation("Player {player} vented to {vent}", e.PlayerControl.PlayerInfo.PlayerName, e.NewVent.Name);
         }
+
+        [EventListener]
+        public void t(IPlayerCompletedTaskEvent e)
```
Comment (Minorusama) [inline src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs]: Change to `OnPlayerCompleteTaskEvent`

File: src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs [hunk 3]
```diff
@@ -111,5 +116,11 @@ public void OnPlayerVentEvent(IPlayerVentEvent e)
...
+        {
+            _logger.LogInformation("Player {player} completed {task},{type},{category} visual {visual}", e.ClientPlayer.Character.PlayerInfo.PlayerName, e.Task.Task.Name, e.Task.Task.Type, e.Task.Task.Category, e.Task.Task.IsVisual);
```
Comment (Minorusama) [inline src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs]: Please fix spacing after commas.
Change `e.ClientPlayer.Character` to `e.PlayerControl`

File: src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs [hunk 4]
```diff
@@ -84,6 +84,11 @@ public async ValueTask OnPlayerChat(IPlayerChatEvent e)
                 await e.PlayerControl.NetworkTransform.SnapToAsync(new Vector2(1, 1));
             }
 
+            if(e.Message == "completetasks")
+            {
+                _ = e.PlayerControl.CompleteAllTasksAsync();
```
Comment (Minorusama) [inline src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs]: Use `await`

**Review by Minorusama — CHANGES_REQUESTED** [review]

**Review by JohnTranQUT — COMMENTED** [review]
All done in 
Commitment: https://github.com/Minorusama/Impostor/pull/4/commits/d31c951b65d5d5c516bd0fef930663235e71cfb2

**Pull request merged**